gax-java/gax/src/main/java/com/google/api/gax/tracing/MetricsTracer.java

```diff
@@ -64,6 +65,7 @@ public class MetricsTracer implements ApiTracer {
 
   public MetricsTracer(MethodName methodName, MetricsRecorder metricsRecorder) {
     this.attributes.put("method_name", methodName.toString());
+    this.attributes.put("language", LANGUAGE);
```

Comment:
nit: Could you document above in Javadocs (maybe in the class javadoc) that `method_name` and `language` will be autopopulated and that the value `Java` for language is by default.

I think it might be interesting if someone has like a kotlin app and uses our library with otel and sees that the language attribute is Java. Granted this is Java SDK and they can override this value themselves if they want.

Reply:
Done. added.